pkg/adapter/containers_remote.go

```diff
@@ -986,3 +986,26 @@ func (r *LocalRuntime) GetNamespaces(container shared.PsContainerOutput) *shared
 	}
 	return &ns
 }
+
+// Commit creates a local image from a container
+func (r *LocalRuntime) Commit(ctx context.Context, c *cliconfig.CommitValues, container, imageName string) (string, error) {
+	var iid string
+	reply, err := iopodman.Commit().Send(r.Conn, varlink.More, container, imageName, c.Change, c.Author, c.Message, c.Pause, c.Format)
+	if err != nil {
+		return "", err
+	}
+	for {
+		responses, flags, err := reply()
+		if err != nil {
+			return "", err
+		}
+		for _, line := range responses.Logs {
+			fmt.Print(line)
```

Comment:
You are writing to os.Stdout here yes? That could break when using bridge.

Reply:
no, that only can happen on the backend.  adapter/ is frontend.  That said, we have talked about long term developing standard ways to print output.

Reply:
This doesn't match how we're doing it for the local client, though. Local client prints only to STDERR, and only if quiet is not passed.

Reply:
I still think this is an issue?